src/main/java/org/mdbenefits/app/cli/TransmissionCommands.java

```diff
@@ -0,0 +1,191 @@
+    }
+
+    private void transmitBatch(List<Transmission> transmissions) {
+
+        Map<String, String> errors = new HashMap<>();
+        log.info("Preparing to send {} transmissions.", transmissions.size());
+
+        transmissions.forEach(transmission -> {
+            Submission submission = transmission.getSubmission();
+            log.info("Sending transmission with ID {} for submission with ID: {}.", transmission.getId(), submission.getId());
+            updateTransmissionStatus(transmission, TransmissionStatus.TRANSMITTING, errors, false);
+            try {
+                byte[] pdfFileBytes = pdfService.getFilledOutPDF(submission);
+
+                String county = (String) submission.getInputData().get("county");
+                String folderId = getCountyFolderId(county);
+                String confirmationNumber = (String) submission.getInputData().get("confirmationNumber");
+
+                String pdfFileName = getPdfFilename(confirmationNumber);
+
+                List<File> existingDirectories = googleDriveClient.findDirectory(confirmationNumber, folderId);
+                if (!existingDirectories.isEmpty()) {
+                    log.info("Found {} existing instances of folder for submission with ID: {}. Deleting existing instances.", existingDirectories.size(),
+                            submission.getId());
+                    // remove folder
+                    for (File dir : existingDirectories) {
+                        googleDriveClient.deleteDirectory(dir.getName(), dir.getId(), errors);
+                    }
+                }
+
+                String entryFolderId = googleDriveClient.createFolder(folderId, confirmationNumber, errors);
+                if (entryFolderId == null) {
+                    // something is really wrong here; note the error and skip the entry
+                    log.error("Failed to create folder for submission with ID: {}.", submission.getId());
+                    updateTransmissionStatus(transmission, TransmissionStatus.FAILED, errors, false);
+                    return;
+                }
+
+                List<UserFile> userFilesForSubmission = userFileRepositoryService.findAllBySubmission(submission);
+
+                for (int count = 0; count < userFilesForSubmission.size(); count++) {
+                    UserFile file = userFilesForSubmission.get(count);
+
+                    // get the file from S3
+                    CloudFile cloudFile = cloudFileRepository.get(file.getRepositoryPath());
+
+                    String fileName = getUserFileName(confirmationNumber, file, count + 1, userFilesForSubmission.size());
+                    log.info("Uploading file {} of {} for submission with ID: {}.", count + 1, userFilesForSubmission.size(),
+                            submission.getId());
+                    // send to google
+                    googleDriveClient.uploadFile(entryFolderId, fileName, file.getMimeType(), cloudFile.getFileBytes(),
```

Comment:
For this and the PDF upload, are we not examining `errors` and updating `Transmission` appropriately?

Reply:
The `errors` are updated in the `uploadFiles` method.  

I fixed the gap around PDF generation and PDF upload